lib/src/mapbox_maps_platform.dart

```diff
@@ -102,8 +105,8 @@ class _MapboxMapsPlatform {
     }
   }
 
-  void initPlatform(String channelSuffix) {
-    this.binaryMessenger = ProxyBinaryMessenger(suffix: channelSuffix);
+  void initPlatform() {
+    this.binaryMessenger = ProxyBinaryMessenger(suffix: "/map_$_channelSuffix");
```

Comment:
but I'm not sure

```suggestion
    this.binaryMessenger = ProxyBinaryMessenger(suffix: "/map_$channelSuffix");
```

Reply:
Not sure I follow - `_channelSuffix` is the variable name (and the resulting string is `/map_0`), do you suggest to rename the variable?

Reply:
ah, okay. I was thinking that second `_` not needed